MASTER FUNCTIONS LIBRARY.vbs

```diff
@@ -5340,6 +5340,73 @@ function convert_digit_to_excel_column(col_in_excel)
+'--- This function standardizes the creation and content for a recertification appointment letter notice
+'~~~~~ programs: The variable to identify the programs applicable to the case
+'~~~~~ CM_plus_1_mo: Current month plus 1 month
+'~~~~~ CM_plus_1_yr: Current month plus 1 year
+'~~~~~ intvw_programs: The variable to identify the program identified during the interview (MFIP/SNAP, MFIP, SNAP)
+'~~~~~ interview_end_date: Last date an interview can be conducted within the recertification window, must be in date format
+'~~~~~ last_day_of_recert: Last day of recertification, must be in date format
+'===== Keywords: MAXIS, ADMIN, NOTICE, Appointment, Recertification
+	CALL write_variable_in_SPEC_MEMO("The Department of Human Services sent you a packet of paperwork. This paperwork is to renew your " & programs & " case.")
+	If len(programs) < 11 Then CALL write_variable_in_SPEC_MEMO("")
+	CALL write_variable_in_SPEC_MEMO("Please sign, date and return the renewal paperwork by " & CM_plus_1_mo & "/08/" & CM_plus_1_yr & ". You must also complete an interview for your " & intvw_programs & " case to continue.")
+	CALL write_variable_in_SPEC_MEMO("")
+	Call write_variable_in_SPEC_MEMO("  *** Please complete your interview by " & interview_end_date & ". ***")
+	Call write_variable_in_SPEC_MEMO("To complete a phone interview, call the EZ Info Line at")
+	Call write_variable_in_SPEC_MEMO("612-596-1300 between 8:00am and 4:30pm Monday thru Friday.")
+	CALL write_variable_in_SPEC_MEMO("")
+	If len(programs) < 11 Then
+		CALL write_variable_in_SPEC_MEMO("**  Your " & programs & " case will close on " & last_day_of_recert & " unless  **")
+	ElseIf len(programs) > 14 Then
+		CALL write_variable_in_SPEC_MEMO("*Your " & programs & " case will close on " & last_day_of_recert & " unless")
+	ElseIf len(programs) > 10 Then
+		CALL write_variable_in_SPEC_MEMO("* Your " & programs & " case will close on " & last_day_of_recert & " unless *")
+	End If
+	CALL write_variable_in_SPEC_MEMO("** we receive your paperwork and complete the interview. **")
+	CALL write_variable_in_SPEC_MEMO("")
+	CALL write_variable_in_SPEC_MEMO("All interviews are completed via phone. If you do not have a phone, go to one of our Digital Access Spaces at any Hennepin County Library or Service Center. No processing, no interviews are completed at these sites. Some Options:")
+	CALL write_variable_in_SPEC_MEMO(" - 7051 Brooklyn Blvd Brooklyn Center 55429")
+	CALL write_variable_in_SPEC_MEMO(" - 1011 1st St S Hopkins 55343")
+	CALL write_variable_in_SPEC_MEMO(" - 1001 Plymouth Ave N Minneapolis 55411")
+	CALL write_variable_in_SPEC_MEMO(" - 2215 East Lake Street Minneapolis 55407")
+	CALL write_variable_in_SPEC_MEMO(" (Hours are 8 - 4:30 Monday - Friday)")
+	Call digital_experience
+	CALL write_variable_in_SPEC_MEMO("Domestic violence brochures are available at this website: https://edocs.dhs.state.mn.us/lfserver/Public/DHS-3477-ENG. You can always request a paper copy via phone.")
```

Comment:
I am a little concerned here because the digital experience function includes a space after the information and this notice is very full and cannot fit if there is a space between the digital experience content and the DV brochure detail.

Reply:
Committed separately and tagged digital experience issue to indicate the space has been removed to accommodate its use with other sections of memo.